<?xml version="1.0"?>

<launch>

  <node pkg="robot_path_planner" name="RobotPathVisualizer" type="RobotPathVisualizer.py" output="screen"/>

</launch>

